### PR TITLE
Paas

### PR DESCRIPTION
**Pull request by hmdesai89**

### CODE REVIEW

File: ec2api/api/account.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        raise exception.DependencyViolation(msg)
+     
+    db_api.delete_item(dummy_context, paas_acc['id'])    
+    return True
+
+def _format_pass_account(context,account, status):
+
+    return {
+        'acc-id' : account,
+        'pass' : status
```
Comment (gdpak) [inline ec2api/api/account.py]: minor typo - pass- should be paas

File: ec2api/api/account.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+CONF = cfg.CONF
+LOG = logging.getLogger(__name__)
+
+
+"""Quota related API implementation
+"""
+
+Validator = common.Validator
+
```
Comment (gdpak) [inline ec2api/api/account.py:36]: Probably add a comment that what is valid account-id ? Is it in form acc-<13-digits>?

File: ec2api/api/cloud.py [hunk 3]
```diff
@@ -1339,6 +1340,30 @@ def update_quota(self, context, account, resource, quota):
...
         """
         """
+    
+    @module_and_param_types(account, 'account_id')
+    def create_paas_account(self, context, account_id):
+        """ Creates a PASS account entry for specified account
+        
+        Args:
+            context (RequestContext): The request context.
+            account_id : ID of the account for which PASS must be enabled
```
Comment (gdpak) [inline ec2api/api/cloud.py]: Please add on what is the format of account-id . What is valid and what is invalid

File: ec2api/api/cloud.py [hunk 4]
```diff
@@ -1339,6 +1340,30 @@ def update_quota(self, context, account, resource, quota):
...
+            Pass account number and pass-id
+        """
+
+    @module_and_param_types(account, 'account_id')
+    def delete_paas_account(self, context, account_id):
+        """ Deletes a PASS account entry for specified account
+        
+        Args:
+            context (RequestContext): The request context.
+            account_id : ID of the account to delete pass
```
Comment (gdpak) [inline ec2api/api/cloud.py]: same as create_account

File: ec2api/api/account.py (hunk 2 above)
Comment (hmdesai89) [reply ec2api/api/account.py:36]: This is same validator which we use while changing quota. It is in acc-<32 byte account id> format.

File: ec2api/api/cloud.py (hunk 3 above)
Comment (hmdesai89) [reply ec2api/api/cloud.py]: acc-<32 byte account id>

**Pull request merged**